CMake/sitkLanguageOptions.cmake

```diff
@@ -15,24 +16,53 @@ sitk_check_dynamic_lookup(MODULE
   SITK_UNDEFINED_SYMBOLS_ALLOWED
   )
 
-option(WRAP_DEFAULT "The default initial value for wrapping a language when it is detected on the system." ON)
+option(WRAP_DEFAULT "The default value for wrapping a language." ON)
```

Comment:
"Enables automatic detection of all languages and enables language wrapping if available."